src/js/base/module/LinkDialog.js

```diff
@@ -56,7 +56,6 @@ export default class LinkDialog {
   bindEnterKey($input, $btn) {
     $input.on('keypress', (event) => {
       if (event.keyCode === key.code.ENTER) {
-        event.preventDefault();
```

Comment:
This line is needed. Please roll back it.

Reply:
I thought I already had added it back in with the last update to this PR